nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -28,6 +28,7 @@ let
   phpPackage = cfg.phpPackage.buildEnv {
     extensions = { enabled, all }:
       (with all; enabled
+        ++ [ bz2 intl sodium ] # recommended
```

Comment:
Small note: I've checked the php-with-extensions derivation used by nextcloud and intl and sodium where already in there but I don't think it hurts listing them twice.